test/tests/functional/pbs_python_restart_settings.py

```diff
@@ -307,7 +307,7 @@ def test_max_hooks(self):
         # every 3s
         logs = self.server.log_match(
             "Restarting Python interpreter to reduce mem usage",
-            allmatch=True, starttime=stime, max_attempts=8)
+            allmatch=True, starttime=stime, max_attempts=8, n='ALL')
```

Comment:
This is not right fix, if you are passing `starttime` you shouldn't need to pass `n`, as if starttime is given then log_match shouldn't limit number of lines, if its limiting number of lines then its bug in log_match